README.md

```diff
@@ -48,14 +48,14 @@ jobs:
 ## Run MATLAB Tests
 When you define your workflow in the `.github/workflows` directory of your repository, specify the **Run MATLAB Tests** action as `matlab-actions/run-tests@v2`.
 
-By default, MATLAB includes any files in your project that have a `Test` label. If your workflow does not use a MATLAB project, or if it uses a MATLAB release before R2019a, then MATLAB includes all tests in the root of your repository and in any of its subfolders.
+By default, MATLAB includes any files in your project that have a `Test` label. If your workflow does not use a MATLAB project, or if it uses a MATLAB release before R2019a, then MATLAB includes all tests in the MATLAB current folder (returned by the [`pwd`](https://www.mathworks.com/help/matlab/ref/pwd.html) command) and in any of its subfolders.
```

Comment:
See my comments on the top-level matlab-actions PR. I don't particularly like tying it to the `pwd` command because the user has to jump through hoops to check that. I also have some concerns about "MATLAB current folder"